monitoring/ceph-mixin/prometheus_alerts.libsonnet

```diff
@@ -672,7 +672,7 @@
           expr: 'ceph_health_detail{name="CEPHADM_FAILED_DAEMON"} > 0',
           labels: { severity: 'critical', type: 'ceph_default', oid: '1.3.6.1.4.1.50495.1.2.1.11.1' },
           annotations: {
-            summary: 'A ceph daemon manged by cephadm is down%(cluster)s' % $.MultiClusterSummary(),
+            summary: 'A ceph daemon managed by cephadm is down%(cluster)s' % $.MultiClusterSummary(),
```

Comment:
I seem to recall a discussion about not wanting to make certain kinds of changes to the prometheus content. Sadly GitHub and my browser are not cooperating atm, so I can't find that in the meta PR.

Offhand, this is a relatively safe change since it's just human facing

Reply:
Maybe we had some unit-test checking the alert message... But I'm fine with this!